ui/admin/app/controllers/scopes/scope/workers/worker/tags.js

```diff
@@ -44,6 +51,40 @@ export default class ScopesScopeWorkersWorkerTagsController extends Controller {
     );
   }
 
+  /**
+   * Determines if a config tag is managed by HCP
+   * @param {object} tag
+   * @returns {boolean}
+   */
+  isHcpManaged(tag) {
```

Comment:
I don't think we should assume those could be the only config values, shouldn't we be checking if we're running in HCP?

Reply:
We cannot simply check if we are running in HCP as other non-hcp managed workers would show the HCP managed tooltip. We will work with cloud team to find a better solution in future releases.